src/post/post.entity.ts

```diff
@@ -302,6 +302,7 @@ export class Post extends BaseEntity {
             escapeHtml: true,
             convertLineBreaks: true,
             wrapInParagraph: true,
+            extractLinks: true,
```

Comment:
We have to add it [here](https://github.com/TryGhost/ActivityPub/blob/main/src/post/post.entity.ts#L350) as well to make it work with replies. 

Optional but I think it will be good to have a small test for `createReply` too.

Reply:
Great catch, thank you!